server/public/styles.css

```diff
@@ -463,6 +492,41 @@ body {
   border-radius: 0.75rem;
 }
 
+.rl-buy,
+.rl-info {
```

Comment:
does `rl` mean reading list? generally speaking, abbreviations are not a good practice, but in this case, it's probably not that bad. Be sure to apply this consistently though, if `rl` means `reading-list` and you have `reading-list` in other places, then `rl` is definitely a WTF moment for anybody reading your code.